packages/runtime/src/components/shared/BackgroundsContainer/index.tsx

```diff
@@ -10,31 +9,54 @@ import {
   useImperativeHandle,
   useState,
 } from 'react'
-import { BackgroundsValue as BackgroundsPropControllerValue } from '../../../prop-controllers/descriptors'
+import {
+  BackgroundsValue as BackgroundsPropControllerValue,
+  ResponsiveNumberValue,
+  ResponsiveSelectValue,
+} from '../../../prop-controllers/descriptors'
 import { useStyle } from '../../../runtimes/react/use-style'
+import { useBoxAnimation } from '../../builtin/Box/animations'
+import { BoxAnimateIn } from '../../builtin/Box/constants'
 import { useBackgrounds } from '../../hooks'
 import Backgrounds from './components/Backgrounds'
 
 type BaseProps = {
-  hasAnimations?: boolean
   backgrounds: BackgroundsPropControllerValue | null | undefined
   children: ReactElement<ElementType>
+  boxAnimateType?: ResponsiveSelectValue<BoxAnimateIn>
+  boxAnimateDuration?: ResponsiveNumberValue
+  boxAnimateDelay?: ResponsiveNumberValue
+  itemAnimationType?: ResponsiveSelectValue<BoxAnimateIn>
 }
 
-type Props = BaseProps & Omit<ComponentPropsWithoutRef<typeof motion.div>, keyof BaseProps>
+type Props = BaseProps & Omit<ComponentPropsWithoutRef<'div'>, keyof BaseProps>
 
 export default forwardRef<HTMLDivElement | null, Props>(function BackgroundsContainer(
-  { hasAnimations = false, backgrounds, children, className, ...restOfProps }: Props,
+  {
+    backgrounds,
+    children,
+    className,
+    boxAnimateType,
+    boxAnimateDuration,
+    boxAnimateDelay,
+    itemAnimationType,
+    ...restOfProps
+  }: Props,
   ref: Ref<HTMLDivElement | null>,
 ) {
   const [handle, setHandle] = useState<HTMLDivElement | null>(null)
-  const Component = hasAnimations ? motion.div : 'div'
 
   useImperativeHandle(ref, () => handle, [handle])
 
+  const animationClassName = useBoxAnimation(
```

Comment:
Could we not have this animation logic live in the Box component and pass the class as a prop to this one? It seems like this component should be unaware of the animation functionality